src/item.h

```diff
@@ -1364,7 +1367,11 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
         * Returns label from "item_label" itemvar and quantity
         */
         std::string label( unsigned int quantity = 0 ) const;
+
     private:
+        /** unique identifier for each non-null instance */
+        item_uid uid_;
```

Comment:
> We aren't using trailing underscores, please don't apply random style elements to the code.

We have recently committed code using both leading and trailing underscores. There is a thread where @Coolthulhu discussed this recently (where?). The leading syntax is a bit questionable due to reserved identifiers, the latter is ugly. Good examples would be the `vitamins1